.github/workflows/module-postgres.yml

```diff
@@ -52,5 +52,5 @@ jobs:
       - name: Test Summary
         uses: test-summary/action@4ee9ece4bca777a38f05c8fc578ac2007fe266f7
         with:
-          paths: "**/TEST-postgres*.xml"
+          paths: "**/TEST-unit*.xml"
```

Comment:
Your are right! https://github.com/testcontainers/testcontainers-go/blob/b76e5fb92637edeb419cf0d934a3d7bdb47176f5/commons-test.mk#L13 the make goal generates the file using unit as variable 🤦‍♂️

Could you apply this change to all the modules, including the template in the module generator? 🙏

Reply:
Done, only the `module-compose.yml` did not need a change as the junitfile was already properly configured. I also adjusted the template in the module generator.

Reply:
I just saw that my linter made changes to the `on` conditions of the workflows. Do you have an idea how to easily fix this as you probably will use a different linter?

Reply:
I'm using vscode + Go plugins. Also golangci-lint, both locally and at CI time. I'd suggest renting back the indents, keeping the jUnit change alone.

Reply:
Prettier was linting the yml files per default on my setup and since the project does not have its own config it just used a default one.